shims/pom.xml

```diff
@@ -30,7 +30,6 @@
 
   <modules>
     <module>common</module>
-    <module>${spark.shim.module}</module>
```

Comment:
Thanks for your following up work!
Do we also need to remove the code where `spark.shim.module`/`spark.test.module` are set.
E.g.:
https://github.com/oap-project/gluten/blob/main/pom.xml#L48
https://github.com/oap-project/gluten/blob/main/pom.xml#L112

Reply:
yeah, removed